frontend/app/src/composables/api/blockchain/accounts.ts

```diff
@@ -45,7 +45,7 @@ function payloadToData({ address, label, tags }: Omit<BlockchainAccountPayload,
     accounts: [
       {
         address,
-        label: label || null,
+        label: isDefined(label) ? label : null,
```

Comment:
wouldn' that also allow '' as a valid label?

Reply:
yes, that's a way to tell backend to remove it from addressbook